x-pack/plugins/observability_solution/slo/public/components/slo/burn_rate/alert_details/components/custom_panels/custom_kql/log_rate_analysis_panel.tsx

```diff
@@ -0,0 +1,285 @@
+        .isAfter(moment(new Date()))
+    ) {
+      return moment(new Date()).valueOf();
+    } else {
+      return alertStart
+        .clone()
+        .add(10 * intervalFactor, 'minutes')
+        .valueOf();
+    }
+  }
+
+  const initialAnalysisStart = {
+    baselineMin: alertStart
+      .clone()
+      .subtract(13 * intervalFactor, 'minutes')
+      .valueOf(),
+    baselineMax: alertStart
+      .clone()
+      .subtract(2 * intervalFactor, 'minutes')
+      .valueOf(),
+    deviationMin: alertStart
+      .clone()
+      .subtract(1 * intervalFactor, 'minutes')
+      .valueOf(),
+    deviationMax: getDeviationMax(),
+  };
+
+  const onAnalysisCompleted = (analysisResults: LogRateAnalysisResultsData | undefined) => {
+    const significantFieldValues = orderBy(
+      analysisResults?.significantItems?.map((item) => ({
+        field: item.fieldName,
+        value: item.fieldValue,
+        docCount: item.doc_count,
+        pValue: item.pValue,
+      })),
+      ['pValue', 'docCount'],
+      ['asc', 'asc']
+    ).slice(0, 50);
+
+    const logRateAnalysisType = analysisResults?.analysisType;
+    setLogRateAnalysisParams(
+      significantFieldValues && logRateAnalysisType
+        ? { logRateAnalysisType, significantFieldValues }
+        : undefined
+    );
+  };
+
+  const messages = useMemo<Message[] | undefined>(() => {
+    const hasLogRateAnalysisParams =
+      logRateAnalysisParams && logRateAnalysisParams.significantFieldValues?.length > 0;
+
+    if (!hasLogRateAnalysisParams) {
+      return undefined;
+    }
+
+    const { logRateAnalysisType } = logRateAnalysisParams;
+
+    const header = 'Field name,Field value,Doc count,p-value';
+    const rows = logRateAnalysisParams.significantFieldValues
+      .map((item) => Object.values(item).join(','))
+      .join('\n');
+
+    const content = `You are an observability expert using Elastic Observability Suite on call being consulted about a log threshold alert that got triggered by a ${logRateAnalysisType} in log messages. Your job is to take immediate action and proceed with both urgency and precision.
+      "Log Rate Analysis" is an AIOps feature that uses advanced statistical methods to identify reasons for increases and decreases in log rates. It makes it easy to find and investigate causes of unusual spikes or dips by using the analysis workflow view.
+      You are using "Log Rate Analysis" and ran the statistical analysis on the log messages which occured during the alert.
+      You received the following analysis results from "Log Rate Analysis" which list statistically significant co-occuring field/value combinations sorted from most significant (lower p-values) to least significant (higher p-values) that ${
+        logRateAnalysisType === LOG_RATE_ANALYSIS_TYPE.SPIKE
+          ? 'contribute to the log rate spike'
+          : 'are less or not present in the log rate dip'
+      }:
+
+      ${
+        logRateAnalysisType === LOG_RATE_ANALYSIS_TYPE.SPIKE
```

Comment:
@simianhacker I was thinking about that. Looks like in our case we are interested only about a spike, when there is an increase in bad events. We don't want to get an analysis if good events start increasing, right?

@walterra It is possible to specify the log rate analysis type in the `LogRateAnalysisContent`, right?

Reply:
Log rate analysis itself analyses the date histogram, checks the baseline+deviation and will determine which type of analysis to run (spike or dip). It's not something we support to override via props until now. Is this page accessible always or only if there is an active alert? If only with active alert, then the "nature" of the alert with something being above a threshold should always result in a "spike" analysis. Since the AI Assistant is something that acts on the analysis results, you could still just hide it if the analysis type would ever be "dip".